docs/kubernetes/ingress-backends.md

```diff
@@ -0,0 +1,70 @@
+# Kubernetes Backend Deployments
+
+## Kubernetes Race Condition problem
+
+As described in [#652](https://github.com/zalando/skipper/issues/652),
+there is a problem that exists in kubernetes, while terminating Pods.
+Terminating Pods could be graceful, but the nature of distributed
```

Comment:
In the Kubernetes docs, 'pod' is written in lowercase. https://kubernetes.io/docs/concepts/workloads/pods/pod/

Reply:
here the do not: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/

Reply:
I think Pod is a name, so I decided to do the first character uppercase. I don't mind, I can change. What do you think?

Reply:
+1 in favor of Pod.

Reply:
\me marked it as resolved :+1: